rs/orchestrator/src/catch_up_package_provider.rs

```diff
@@ -71,15 +75,16 @@ impl CatchUpPackageProvider {
         registry: Arc<RegistryHelper>,
         cup_dir: PathBuf,
         crypto: Arc<dyn ThresholdSigVerifierByPublicKey<CatchUpContentProtobufBytes> + Send + Sync>,
+        crypto_tls_config: Arc<dyn TlsConfig + Send + Sync>,
```

Comment:
Can't you combine them `dyn X+Y`?

Reply:
I could not combine in a simple way:

```
pub(crate) trait Crypto:
    ThresholdSigVerifierByPublicKey<CatchUpContentProtobufBytes> + TlsConfig + Send + Sync
{
}
impl<T> Crypto for T where
    T: ThresholdSigVerifierByPublicKey<CatchUpContentProtobufBytes> + TlsConfig + Send + Sync
{
}
```

Looks more confusing to me than having an extra parameter